### PR TITLE
Fix broken docs links for Networks

### PR DESCRIPTION
**Pull request by catlook**
also fixed link in conf.py

Closes issue #438

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-26f768ba32/docs/user-guides/infrastructure/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-26c76e5c24/docs/user-guides/infrastructure/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-e20c961ac9/docs/user-guides/infrastructure/).

File: cloud-interfaces/api/cloudnetworks-api.rst [hunk 1]
```diff
@@ -145,27 +145,24 @@ In the sample response,
...
-<networks/api/v2/cn-gettingstarted/content/neutron_list_networks_curl.html>`.
+:rax-docs:`Getting Started Guide for the Cloud Networks API <cloud-networks/v2/getting-started/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing networks (cURL)
+<cloud-networks/v2/getting-started/managing-networks/creating-network-curl/#listing-networks-curl>`.
 
 Learn about Cloud Networks in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 In the
 :rax-dev-quickstart:`SDK QuickStart for Cloud Networks <cloud-networks/getting-started>`,
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudnetworks-api.rst]: This link gives a 404. (And there is the same link below in the paragraph starting with "To see examples" that also gives a 404

File: cloud-interfaces/api/cloudservers-api.rst [hunk 2]
```diff
@@ -115,16 +115,16 @@ for tag, status, image, flavor, and type.
 Learn about Cloud Servers in API documentation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 In the
-:rax-api:`API cross-reference <>`,
+:rax-docs: `Developer documentation <>`.
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudservers-api.rst]: This link is broken - it looks like this in the output: :rax-docs: Developer documentation <>

File: cloud-interfaces/api/cloudservers-api.rst [hunk 3]
```diff
@@ -155,17 +155,16 @@ the request parameters named ``status``, ``image``, and ``flavor``
...
-demonstrates
-:rax-docs:`obtaining a detailed list of Cloud Servers by using the cURL command-line interface (CLI)
-<servers/api/v2/cs-gettingstarted/content/curl_list_servers.html>`.
+and walks you through basic server and network operations.
+An intermediate step shows an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing servers (cURL)
+<cloud-servers/v2/getting-started/create-server/listing-servers-curl/>`.
 
 Learn about Cloud Servers in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudservers-api.rst]: There are 2 links in this section to SDK QuickStart for Cloud Servers that gives a page not found..

File: cloud-interfaces/api/cloudnetworks-api.rst [hunk 4]
```diff
@@ -108,16 +108,16 @@ you can see
 Learn about Cloud Networks in API documentation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 In the
-:rax-api:`API cross-reference <>`,
+:rax-docs: `Developer documentation <>`.
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudnetworks-api.rst]: Broken link - it looks like this in output: :rax-docs: Developer documentation <>.

File: cloud-interfaces/api/cloudimages-api.rst [hunk 5]
```diff
@@ -142,10 +141,10 @@ In the sample response,
...
-you can see an example of
-:rax-docs:`obtaining a list of images by using the cURL command-line interface (CLI)
-<images/api/v2/ci-gettingstarted/content/list-images.html>`.
+:rax-docs:`Getting Started Guide for the Cloud Images <cloud-images/v2/getting-started/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing images
+<cloud-images/v2/getting-started/use-images/#listing-images>`.
 
 Learn about Cloud Images in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudimages-api.rst]: There is a link below this to the SDK QuickStart for Cloud Images that gives a 404... and one at the end of the section as well.

File: cloud-interfaces/api/cloudfiles-api.rst [hunk 6]
```diff
@@ -115,10 +116,10 @@ In the sample response, ``bytes`` and ``name`` correspond
...
-you can see an example of
-:rax-docs:`obtaining a list of account details and Cloud Files containers by using the cURL command-line interface (CLI)
-<files/api/v1/cf-getting-started/content/Show-Account-Details-d1e101.html>`.
+<cloud-files/v1/getting-started/use-API-directly/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Creating a storage container
+<cloud-files/v1/getting-started/use-API-directly/#creating-a-storage-container>`.
 
 Learn about Cloud Files in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudfiles-api.rst]: There are 2 links to SDK QuickStart for Cloud Files in this section that give a page not found message.

File: cloud-interfaces/api/cloudblockstorage-api.rst [hunk 7]
```diff
@@ -159,10 +158,10 @@ In the sample response,
...
-you can see an example of
-:rax-docs:`obtaining a list of Cloud Block Storage volumes by using the cURL command-line interface (CLI)
-<cbs/api/v1.0/cbs-getting-started/content/Listing_volumes_d1e060.html>`.
+:rax-docs:`Getting Started Guide for the Cloud Block Storage API <cloud-block-storage/v1/getting-started/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing existing block storage volumes
+<cloud-block-storage/v1/getting-started/use-API-directly/#listing-existing-block-storage-volumes>`.
 
 Learn about Cloud Block Storage in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (cyrichardson) [inline cloud-interfaces/api/cloudblockstorage-api.rst]: There are 2 broken links in this section to SDK QuickStart for Cloud Block Storage

File: cloud-interfaces/api/cloudnetworks-api.rst (hunk 4 above)
Comment (catlook) [inline cloud-interfaces/api/cloudnetworks-api.rst]: done

File: cloud-interfaces/api/cloudnetworks-api.rst (hunk 1 above)
Comment (catlook) [inline cloud-interfaces/api/cloudnetworks-api.rst]: interesting.  the links aren't broken when i used the nexus preview of this commit.  leaving as is for now,

Comment (cyrichardson) [inline cloud-interfaces/api/cloudnetworks-api.rst]: I used the preview link to review/check links, too.

File: cloud-interfaces/api/cloudimages-api.rst (hunk 5 above)
Comment (catlook) [inline cloud-interfaces/api/cloudimages-api.rst]: those were definitely broken, though i swear i tested them.  should be fixed now

File: cloud-interfaces/api/cloudservers-api.rst (hunk 3 above)
Comment (catlook) [inline cloud-interfaces/api/cloudservers-api.rst]: those were definitely broken, though i swear i tested them. should be fixed now

File: cloud-interfaces/api/cloudservers-api.rst (hunk 2 above)
Comment (catlook) [inline cloud-interfaces/api/cloudservers-api.rst]: fixed

File: cloud-interfaces/api/cloudimages-api.rst (hunk 5 above)
Comment (catlook) [inline cloud-interfaces/api/cloudimages-api.rst]: think these are ok

File: cloud-interfaces/api/cloudfiles-api.rst (hunk 6 above)
Comment (catlook) [inline cloud-interfaces/api/cloudfiles-api.rst]: think these are ok

File: cloud-interfaces/api/cloudblockstorage-api.rst (hunk 7 above)
Comment (catlook) [inline cloud-interfaces/api/cloudblockstorage-api.rst]: think these are ok

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-eb8f7f3655/docs/user-guides/infrastructure/). :bow:

**Pull request merged**